app/js/dash/DashManifestExtensions.js

```diff
@@ -527,7 +527,9 @@ Dash.dependencies.DashManifestExtensions.prototype = {
         // TODO The client typically should not use the time at which it actually successfully received the MPD, but should
         // take into account delay due to MPD delivery and processing. The fetch is considered successful fetching
         // either if the client obtains an updated MPD or the client verifies that the MPD has not been updated since the previous fetching.
-        var fetchTime = this.timelineConverter.calcPresentationTimeFromWallTime(manifest.loadedTime, period);
+        // note that if the loadedTime has been modified (which would have happened if time sync was completed), we will need to ignore
+        // the offset in the time calculation. bit of a hack, but makes things easier everywhere else loadTime is used.
+        var fetchTime = this.timelineConverter.calcPresentationTimeFromWallTime(manifest.loadedTime, period, manifest.loadedTimeModified);
```

Comment:
conceptually there are two load times we care about - the client wall clock of the load and the server wall clock of the load. (we do not care about wall clocks of each of the CDN endpoints and encoders and servers in the path, just that one to which AvailabilityStartTime is mapped)
There should never be a 'modified' load time. In dash.js only the client time is stored and then an offset is calculated and stored to map the client wall clock to the server wall clock so that the client wall clock times can be mapped to and from AvailabilityStartTime.